Cython/Utility/ObjectHandling.c

```diff
@@ -1992,6 +1992,26 @@ typedef struct {
 /////////////// UnpackUnboundCMethod ///////////////
 //@requires: PyObjectGetAttrStr
 
+#define __PYX_REINTERPRET_FUNCION(func_pointer, other_pointer) ((func_pointer)(void(*)(void))(other_pointer))
+#define __PYX_REINTERPRET_POINTER(pointer_type, pointer) ((pointer_type)(void *)(pointer))
+#define __PYX_RUNTIME_REINTERPRET(type, var) (*(type *)(&var))
+
+static PyObject *__Pyx_SelflessCall(PyObject *method, PyObject *args, PyObject *kwargs) {
+    PyObject *selfless_args = PyTuple_GetSlice(args, 1, PyTuple_Size(args));
```

Comment:
Do I need to use cython versions of those, if they exist (I didn't really check)?

Reply:
No, I don't think so - they're straight-forward and universally available so there's nothing for Cython to wrap.